assets/app/scripts/controllers/build.js

```diff
@@ -7,13 +7,16 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('BuildController', function ($scope, $routeParams, DataService, project, BuildsService, $filter) {
+  .controller('BuildController', function ($scope, $routeParams, DataService, ProjectsService, BuildsService, $filter) {
+    $scope.projectName = $routeParams.project;
     $scope.build = null;
     $scope.buildConfigName = $routeParams.buildconfig;
     $scope.builds = {};
     $scope.alerts = {};
-    $scope.renderOptions = $scope.renderOptions || {};
-    $scope.renderOptions.hideFilterWidget = true;
+    $scope.renderOptions = {
+      hideFilterWidget: true,
+      showSidebarRight: false
```

Comment:
whats the default behavior for showSidebarRight, im tempted to make false the default since we are having to set it to false all over the place now